truckersmp-cli

```diff
@@ -91,6 +94,25 @@ class AppId:
     proton = {}
 
 
+def check_libsdl2():
+    """
+    Check whether SDL2 shared object file is present.
+
+    If SDL2 is detected, this function returns True.
+    Otherwise, this returns False.
+
+    Currently we check it only on Linux systems.
```

Comment:
Because this is a general description this should probably in neutral form. Something like:
> Currently the check only performs on…
> The check only runs on…

Reply:
Changed to "Currently the check is done only on Linux systems."